src/styles.css

```diff
@@ -73,7 +73,7 @@ body {
 
 .board-row:after {
   clear: both;
-  content: '';
+  content: "";
```

Comment:
Keep the single quotes. For more info check the Google CSS Styling Guide: https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks

Reply:
I see, thank you so much for the resources!